packages/desktop/src/renderer/components/Settings/Tabs/Attachments/Attachments.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { type FC, useCallback } from 'react'
+import { useDispatch, useSelector } from 'react-redux'
+
+import { settings } from '@quiet/state-manager'
+
+import { AttachmentsComponent } from './AttachmentsComponent'
+
+interface useAttachmentsDataReturnType {
+  maxAutoDownloadBytes: number
+}
+
+export const useAttachmentsData = (): useAttachmentsDataReturnType => {
+  console.log(settings.selectors)
+  console.log(settings.selectors.getMaxAutodownloadBytes)
+  const data = {
+    maxAutoDownloadBytes: useSelector(settings.selectors.getMaxAutodownloadBytes),
+  }
+  return data
+}
+
+export const useAttachmentsActions = (_maxAutoDownloadBytes: number) => {
+  const dispatch = useDispatch()
+
+  const setMaxAutodownloadBytes = useCallback(
+    (option: number) => {
+      dispatch(settings.actions.setMaxAutodownloadBytes(option))
+    },
+    [dispatch]
+  )
+
+  return { setMaxAutodownloadBytes }
+}
+
+export const Attachments: FC = () => {
+  const { maxAutoDownloadBytes } = useAttachmentsData()
+  const { setMaxAutodownloadBytes } = useAttachmentsActions(maxAutoDownloadBytes)
```

Comment:
Our code style doesn't tend to wrap these
```suggestion
  const dispatch = useDispatch()

  const setMaxAutodownloadBytes = useCallback(
    (option: number) => {
      dispatch(settings.actions.setMaxAutodownloadBytes(option))
    },
    [dispatch]
  )

```